### ISSUE TITLE
Improve accessible output from Link List content template when image not selected

### ISSUE THREAD

**Issue by CrispinF** [cloudscribe/cloudscribe.SimpleContent#459]
This render view https://github.com/cloudscribe/cloudscribe.SimpleContent/blob/master/src/cloudscribe.SimpleContent.ContentTemplates.Bootstrap4/Views/Shared/LinksRenderPartial.cshtml 
outputs empty links if no image is specified for a link list item. This fails accessibility testing.
(Also it would be useful to assign a css class to each element e.g. "linklist-item-description".)

**Comment by CrispinF** [comment]
@joeaudette thanks! - and can this also go into the core 3 branch?
